tests/unit/test_serialization.py

```diff
@@ -401,12 +471,22 @@ def assert_single_dataset_entries(
         # Complete component data checks
         if is_columnar_filter(data_filter, component):
             assert isinstance(deserialized_output, dict)
+            # Assert number of batches are equal for all attributes
             assert all(len(v) == len(serialized_input) for v in deserialized_output.values())
+            # Assert number of attributes are not more than input for all batches
+            # Discrepencies in equal / less than input captured in individual check
+            # assert all(len(v) >= len(deserialized_output) for v in serialized_input)
```

Comment:
forgotten comment?

```suggestion
            # Assert number of attributes are not more than input for all batches
            # Discrepencies in equal / less than input captured in individual check
            assert all(len(v) >= len(deserialized_output) for v in serialized_input)
```

Reply:
This is the test statement where the new 2 variations of ellipsis can be tested. 
Ill mark it as TODO.